lib/resources/example_json/soarian_examples_observation.rb

```diff
@@ -0,0 +1,161 @@
+                        }
+                    ],
+                    "text": "Light Tobacco Smoker"
+                }
+            }
+        },
+        {
+            "resource": {
+                "resourceType": "Observation",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;ul ccdRefId=\"ZSocial_History.Social_History.117449.86444.value.0\">&lt;li>Patient Birth Sex : Unknown&lt;/li>&lt;/ul>"
+                },
+                "identifier": [
+                    {
+                        "use": "official",
+                        "value": "ZSocial_History.Social_History.117449.86444.value.0"
+                    }
+                ],
+                "status": "final",
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://loinc.org",
+                            "code": "76689-9",
+                            "display": "Patient_BirthSex"
+                        }
+                    ],
+                    "text": "Patient_BirthSex"
+                },
+                "subject": {
+                    "reference": "Patient/FCC941D7-60B9-491D-BEED-27629E47CD4E"
+                },
+                "effectiveDateTime": "2017-04-21",
+                "valueCodeableConcept": {
+                    "coding": [
+                        {
+                            "system": "http://hl7.org/fhir/v3/NullFlavor",
+                            "code": "UNK",
+                            "display": "Unknown"
+                        }
+                    ],
+                    "text": "Unknown"
+                }
+            }
+        },
+        {
+            "resource": {
+                "resourceType": "Observation",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;ul ccdRefId=\"ZVitalSigns.Vitals.121415.91009\">&lt;li>A_Temperature : 97 F&lt;/li>&lt;/ul>"
+                },
+                "identifier": [
+                    {
+                        "use": "official",
+                        "value": "ZVitalSigns.Vitals.121415.91009"
+                    }
+                ],
+                "status": "final",
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://loinc.org",
+                            "code": "8310-5",
+                            "display": "A_Temperature"
+                        }
+                    ],
+                    "text": "A_Temperature"
+                },
+                "subject": {
+                    "reference": "Patient/FCC941D7-60B9-491D-BEED-27629E47CD4E"
+                },
+                "effectiveDateTime": "2017-05-23",
+                "valueQuantity": {
+                    "value": 36.111154,
+                    "unit": "Cel",
+                    "system": "http://unitsofmeasure.org/"
```

Comment:
We will consider in future release to remove the trailing slash as it needs code change and we are too close to certification date.